corehq/apps/reports/standard/monitoring.py

```diff
@@ -1428,9 +1428,11 @@ def _rows_by_user(self, report_data):
         last_form_by_user = self.es_last_submissions()
         for user in self.users_to_iterate:
             active_cases = int(report_data.active_cases_by_owner.get(user["user_id"].lower(), 0)) + \
-                sum([int(report_data.active_cases_by_owner.get(group_id, 0)) for group_id in user["group_ids"]])
+                sum([int(report_data.active_cases_by_owner.get(group_id, 0)) for group_id in user["group_ids"]]) + \
```

Comment:
I looked again at the code and it does appear to count cases from other groups the users belong to, so perhaps the correct behavior is to include the location cases here as well. Either way I will have to change the way the total is calculated since it is double counting anyway.